pipeline_steps/checkmarx.groovy

```diff
@@ -24,7 +24,8 @@ def scan(String scan_type, String repo_name, String exclude_folders){
             throw new Exception("Invalid scan type: ${scan_type}, should be default or pci")
         }
         // This step has a habit of throwing NPEs, retry it. RE
-        retry(3) {
+        waitTime = 30
```

Comment:
This could potentially timeout the job, as the max wait time is 8.5 hrs.

I suggest starting at 8s, then the max wait time is ~ 2.25 hours. 

```
In [19]: def exwait(wait, tries):
    ...:     total=0
    ...:     wait /= 2
    ...:     for _ in range(tries):
    ...:         wait *= 2
    ...:         total += wait
    ...:     return total
    ...:
    ...:

In [20]: exwait(30,10) / (60*60.0)
Out[20]: 8.525

In [21]: exwait(8,10) / (60*60.0)
Out[21]: 2.2733333333333334

In [22]: exwait(4,10) / (60*60.0)
Out[22]: 1.1366666666666667

```

```suggestion
        waitTime = 8
```